feat(review-apps): Add formation to Documentation (#2619)
Scalingo/documentation: src/_posts/platform/app/2000-01-01-app-manifest.md

}
```

## Container Formation

> The `scalingo.json` manifest can be used to setup a container formation. A container formation defines any type of container that will be launched on startup.
> 

Reviewer: ```suggestion
The `scalingo.json` manifest can be used to setup a container formation. A container formation defines the shape of the containers that will be launched.

```

`type` seems too vague, shape or size is more adequate IMO.
Furthermore, the word `on startup` doesn't seems to add anything, isn't it?

Author: Your true :), I commit the suggestion

### Review Apps

> You can specify the number of containers of each type to be launched when a Review App is created. This is useful for making an exact copy of a production environment in Review Apps without having to perform any manual actions.

Reviewer: ```suggestion
You can specify the shape and the number of containers to be launched when a Review App is created. This is useful for making an exact copy of a production environment in Review Apps without having to perform any manual actions.
```
